app/src/sharedTest/java/org/oppia/android/app/topic/revisioncard/RevisionCardFragmentTest.kt

```diff
@@ -125,7 +126,7 @@ class RevisionCardFragmentTest {
       )
     ).use {
       openActionBarOverflowOrOptionsMenu(context)
-      onView(withText(context.getString(R.string.help))).perform(ViewActions.click())
+      onView(withText(context.getString(R.string.help_activity_title))).perform(ViewActions.click())
```

Comment:
Use `help_menu` here as it is part of menu.

Reply:
changed help_activity_title to menu_help